src/App.css

```diff
@@ -171,3 +171,9 @@ iframe .header-container .location-btn {
     zoom: 90%;
   }
 } */
+.card-with-image:hover {
+  background-color: #77aede;
+}
+.card-with-image:hover .font-color {
+  color: #fff !important;
```

Comment:
try not to use important.

Reply:
yes @nikhil2128  but in this project maximum css are inline so that we  used important somewhere